auth/src/main/java/com/vimeo/networking2/Authenticator.kt

```diff
@@ -117,50 +125,90 @@ interface Authenticator {
         email: String,
         password: String,
         marketingOptIn: Boolean,
-        authCallback: VimeoCallback<VimeoAccount>
+        callback: VimeoCallback<VimeoAccount>
     ): VimeoRequest
 
     /**
-     * Login via email.
+     * Login via email to obtain a logged in token.
      *
-     * @param email         Email address associated with your Vimeo account.
-     * @param password      Password for your Vimeo account.
+     * @param email Email address associated with your Vimeo account.
+     * @param password Password for your Vimeo account.
+     * @param callback Callback to be notified of the result of the request.
      *
      * @return A [VimeoRequest] object to cancel API requests.
      */
     fun emailLogin(
         email: String,
         password: String,
-        authCallback: VimeoCallback<VimeoAccount>
+        callback: VimeoCallback<VimeoAccount>
     ): VimeoRequest
 
+    /**
+     * Exchange an OAuth 1 token and secret for a new OAuth 2 token.
+     *
+     * @param token The old token to use in the exchange.
+     * @param tokenSecret The old token secret to use in the exchange.
+     * @param callback Callback to be notified of the result of the request.
```

Comment:
It's redundant since the type is clearly visible in the docs that display on hover. "Callback" is an accurate description of what it is. 

<img width="320" alt="Screen Shot 2020-08-04 at 2 44 55 PM" src="https://user-images.githubusercontent.com/1966779/89332325-37873100-d661-11ea-9b73-cca11aa5e9ea.png">